src/components/sbb-slider/sbb-slider.scss

```diff
@@ -0,0 +1,148 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include host-component-properties;
+
+:host {
+  --sbb-slider-line-height: #{px-to-rem-build(4)};
```

Comment:
What confuses me is, that if you use display: inline, it still takes full width and following text is on the next line. I'd expect that text comes directly after the slider (as it would be with inline-block). But maybe it's ok for this case, but then maybe it can be only display: block from default values, as it doesn't seem to make any difference

Reply:
@jeripeierSBB probably the best solution is to keep display: inline-block but also move these

```suggestion
  max-width: #{px-to-rem-build(400)};
  min-width: #{px-to-rem-build(150)};
  width: 100%;
```

from .sbb-slider__wrapper to :host

in this way it behaves as a inline-block element and still has correct dimensions

Reply:
But then it's again 100% width as defined, and we should avoid defining width on host. So maybe there is no perfect solution, or I just accept that it is a block element.
Other question: why is width 100% needed on host (if you would change) ?

Reply:
in this case width 100% is needed to make the element shrink between min and max width, otherwise it sets to the minimum (150px)

Reply:
Yes but this is what I'd expect and the consumer has to set the width he likes. If there is no width, take minimum. in form-field then it should be 100%

Reply:
Also the form-field element has min and max width already set to 150 and 300
Can the consumer eventually overrides max-width property?

Reply:
@federicoisepponfincons In your solution, you would then apply display: inline-block to the host?

Reply:
The more I think, I guess Federicos idea is a good solution that the width can be set directly outside, otherwise we should provide css vars. But anyways, style on host should always be considered carefully :)